odxtools/compumethods/createanycompumethod.py

```diff
@@ -184,6 +180,7 @@ def create_any_compu_method_from_et(et_element: ElementTree.Element,
         return TabIntpCompuMethod(
             internal_points=internal_points, physical_points=physical_points, **kwargs)
 
-    # TODO: Implement other categories (never instantiate CompuMethod)
-    logger.warning(f"Warning: Computation category {compu_category} is not implemented!")
+    # TODO: Implement all categories (never instantiate the CompuMethod base class!)
+    odxraise(f"Warning: Computation category {compu_category} is not implemented!")
```

Comment:
Blow the whole thing with odxraise since something is not implemented?

Reply:
yes. the idea is that if you would like to try anyway, you should give the non-strict mode a shot.